tests/Tests/100__deploy/130__operators/130__rhods_operator/132__traefik-proxy_restart_verification.robot

```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Documentation       132 - RHODS_TRAEFIK_PROXY_CONTAINER_RESTART_VERIFICATION
+...                 Verify that rhods traefik-proxy container
+...                 restart verification
+...
+...                 = Variables =
+...                 | Namespace         | Required |    RHODS Namespace/Project for RHODS operator POD |
+...                 | LABEL_SELECTOR    | Required |    Label selector for traefik proxy|
+
+Resource            ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+
+
+*** Variables ***
+${NAMESPACE}            redhat-ods-applications
+${LABEL_SELECTOR}       name=traefik-proxy
+
+
+*** Test Cases ***
+Verify Traefik Proxy Containers Restart
```

Comment:
What about `Verify Traefik Proxy Containers Have Zero Restarts`?

Reply:
I like this